osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyCalculator.cs

```diff
@@ -258,6 +268,29 @@ private double computeFlashlightRating(double flashlightDifficultyValue, Mod[] m
             return flashlightRating * Math.Sqrt(ratingMultiplier);
         }
 
+        /// <summary>
+        /// Calculates the average amount of score per object that is caused by all sorts of slider ticks.
+        /// </summary>
+        private double calculateSliderNestedScorePerObject(IBeatmap beatmap, int objectCount)
```

Comment:
Shouldn't this be in the `OsuLegacyScoreSimulator`?

Reply:
This isn't something the simulator uses, and changing simulator feels outside of our scope (PP). Given the other comment about moving methods out, I'll probably create some utils class to prevent this stuff from getting out of hand.

Reply:
Moved to `LegacyScoreUtils` in b7b4705fa252bb7b96606f439f179f7a8ba2fc2d